src/App.js

```diff
@@ -1,27 +1,12 @@
-import logo from './logo.svg';
+import React from 'react';
+import Calculator from './components/Calculator';
 import './App.css';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
```

Comment:
Kindly make sure that:
- [ ]  The App.js(main component) has the imported Calculator.js component.
- [ ] The `App.js` component should is class-based.

Reply:
All the suggested changes have been applied :heavy_check_mark: